server/src/completions/variable-flags.ts

```diff
@@ -0,0 +1,12 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export const VARIABLE_FLAGS = [
+  'cleandirs',
+  'depends',
+  'deptask',
+  'dirs',
+  'doc'
```

Comment:
there are more flags than this, like `noexec`. Could you extract this list from a documentation somewhere?

Reply:
scan added